docs/cookbook/recipe_sortable_listing.rst

```diff
@@ -104,42 +105,31 @@ Now you can update your ``services.yaml`` to use the handler provided by the ``p
         app.admin.client:
             class: App\Admin\ClientAdmin
             tags:
-                - { name: sonata.admin, model_class: App\Entity\Client, controller: 'PixSortableBehaviorBundle:SortableAdmin', manager_type: orm, label: 'Clients' }
+                - { name: sonata.admin, model_class: App\Entity\Client, controller: 'Runroom\SortableBehaviorBundle\Controller\SortableAdminController', manager_type: orm, label: 'Clients' }
```

Comment:
^0.16 won't require this change. IMO we should wait for it before merging this. I hope is one of the latest BC break I have to apply.

Reply:
Changed to `RunroomSortableBehaviorBundle:SortableAdmin`

Reply:
@jordisala1991  any eta for the release ? Cause we could also merge this and update it later.

Reply:
@jordisala1991 Runroom\SortableBehaviorBundle don't require FQCN syntax directly. I think we can left short syntax for now

Reply:
What I mean, is that on:

^0.15, you need to add this controller just like you did 👍 
^0.16, you won't need to change the default controller, it will work without it.

Reply:
with this change: https://github.com/Runroom/runroom-packages/pull/133

Reply:
It 100% BC break, because SortableAdminController renamed to MoveAction. May be to rollout a new major release?

Reply:
That's why I use 0., but normally I don't do this kind of BC breaks, specially on the sortable bundle. But IMO in this case is justified, it simplifies a lot the configuration, and allows you to extend the crud controller for another things, without having to deal with the SortableController.

Reply:
If you mean 1.0.0, Not planned atm. But BC breaks are taken into acount, and when they happen I release a new minor just like I would do it with 1.0.0, 2.0.0, but I prefer to be on 0.15 rather than on 15.0.0.

Reply:
- Added `at least version ^0.16`
- Changed `controller` to `SonataAdminBundle:CRUD`

Reply:
@jordisala1991 Should we wait for stable v0.16 ?

Reply:
IMO Yes, it will be ready next week.